apps/andi/test/unit/andi/input_schemas/ingestion_test.exs

```diff
@@ -18,7 +18,7 @@ defmodule Andi.InputSchemas.Ingestion.IngestionTest do
     schema: [
       %{id: Ecto.UUID.generate(), name: "name", type: "type", bread_crumb: "name", dataset_id: "id", selector: "/cam/cam"}
     ],
-    sourceFormat: "sourceFormat"
+    sourceFormat: "text/csv"
```

Comment:
Leaving this change and the one below because they are both improvements to existing tests, even though the file under test isn't changed anymore